app/client/src/constants.js

```diff
@@ -13,11 +13,11 @@ angular.module('reg')
     .constant('DASHBOARD', {
         UNVERIFIED: 'You should have received an email asking you verify your email. Click the link in the email and you can start your application!',
         INCOMPLETE_TITLE: 'You still need to complete your application!',
-        INCOMPLETE: 'If you do not complete your application before the [APP_DEADLINE], you will not be considered for the admissions lottery!',
+        INCOMPLETE: 'If you do not complete your application before the [APP_DEADLINE], you will not be considered for admissions!',
         SUBMITTED_TITLE: 'Your application has been submitted!',
         SUBMITTED: 'Feel free to edit it at any time. However, once registration is closed, you will not be able to edit it any further.\nPlease make sure your information is accurate before registration is closed!',
-        CLOSED_AND_INCOMPLETE_TITLE: 'Unfortunately, registration has closed, and the lottery process has begun.',
-        CLOSED_AND_INCOMPLETE: 'Because you have not completed your profile in time, you will not be eligible for the lottery process.',
+        CLOSED_AND_INCOMPLETE_TITLE: 'Unfortunately, registration has closed, and the admissions process has begun.',
+        CLOSED_AND_INCOMPLETE: '',
```

Comment:
let's just remove this string and all references of it from the template

Reply:
Do you want me to get rid of any reference to CLOSED_AND_INCOMPLETE variable from the app? Will it not be helpful to just have the variable but give it an empty string just incase we want to use the variable later on?

Reply:
good point, could you replace it with a suitable placeholder text for now?